packages/@aws-cdk/aws-pipes-sources-alpha/test/__snapshots__/dynamodb.test.ts.snap

```diff
@@ -0,0 +1,68 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
What is this snapshot file?

Reply:
It's a result of this test: https://github.com/aws/aws-cdk/pull/29476/files#diff-1196955de9d57d66bf83a476e71951d39e5e2735a0eda43541d7842e80bef677R187-R212.